src/__ tests __/Search.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
I find it more convenient to have unit test files in the same directory as the component - so just keep this option in mind and feel free to update the file structure if you'd like